src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -426,6 +432,26 @@ public String actualPathForPublicPath(String publicPath) {
         return null;
     }
 
+    /**
+     * Checks whether a path (defined by a URI) contains a certain sub-path (defined by a URI)).
```

Comment:
optional mini-correction:
```suggestion
     * Checks whether a path (defined by a URI) contains a certain sub-path (defined by a URI).
```